delta-lake/common/src/main/delta-io/scala/com/nvidia/spark/rapids/delta/shims/ShimDeltaUDF.scala

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2022, NVIDIA CORPORATION. All rights reserved.
+ * Copyright (c) 2023, NVIDIA CORPORATION.
```

Comment:
I'm guessing this is ok since I think this is really new file and git got confused, right?  otherwise 2022-2023

Reply:
Yes, this is a tiny file that is essentially new but git thinks it's a rename.